currency/resetwhenleave.cc.lua

```diff
@@ -0,0 +1,11 @@
+{{/* 
+    This Will Delete The Balance of The User That is Leaving 
+
+    Put This In The user Leave Message Notification
+
+*/}}
+
+{{ dbDel .User.ID "HAND" }}
+{{ dbDel .User.ID "NETWORTH" }}
+{{ dbDel .User.ID "GBANK" }}

```

Comment:
Names need to be configurable, extract these to configuration variables

Reply:
done